drivers/focuser/rainbowRSF.h

```diff
@@ -76,6 +77,8 @@ class RainbowRSF : public INDI::Focuser
 
         const static uint32_t homePosition { 8000 };
 
+        // Timer used to timeout when waiting for requested movement to complete.
+        std::unique_ptr<INDI::ElapsedTimer> m_MovementTimer;
```

Comment:
There is no need to use unique_tr, you can just declare it as INDI::ElapsedTimer m_MovementTimer. The class internally manages its own smart pointer.